### DISCUSSION TITLE
I don't know why but I can't do anything with termux

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-app#2244] [Q&A] [answered]
In this days I'm trying to install Kali Linux on my old phone. I always have the same error when I try to install pkg or make an apt-update: "the repository "URL" does not have a release file" or "see apt-secure manpage for repository creation and user configuration" and I don't understand what is happening. I hope someone can help me, thank you guys for your attention. Have a Nice Day

**Comment by agnostic-apollo** [comment, marked as answer]
termux/termux-packages#6726 for > android `7` or #2115 if using android `5/6`.

**Reply by ghost** [reply]
But the #2115 talks about android 6.0

**Reply by agnostic-apollo** [reply]
How am I supposed to know what android version is "my old phone" since you haven't posted any device or termux app version details. Use info in either of the links to fix your issue depending on your android version.

**Reply by ghost** [reply]
> #2115 talks about android 6.0

Everything applicable to Android 6.0 is also applicable to 5.0. Things applicable to Android 7.0 can be used only on Android 7.0 or higher.
